loadtest/user/userentity/actions.go

```diff
@@ -1385,6 +1393,11 @@ func (ue *UserEntity) UpdateThreadFollow(teamId, threadId string, state bool) er
 	return err
 }
 
+// UpdateThreadLastUpdateAt updates the lastUpdateAt of the given thread
```

Comment:
Do we really need this wrapper to simply set a store value? This layer was mostly meant for API calls (+ store as needed).

Reply:
I resorted to this because of our UserStore/MutableUserStore abstraction. The u.Store() returns store.Userstore which does not include writable methods. Do we want to change that?

Reply:
Oh man, that's something I had forgotten about. I guess I really tried to avoid mutations from the higher level. Maybe a small comment with this rationale is sufficient for now. If we start seeing a rising need for mutating in the controller then we may reconsider and expose the whole thing.

Reply:
Haha okay.